.env.example

```
@@ -28,4 +22,21 @@ DATABASE_URL="postgresql://postgres:password@localhost:5432/bootstrap"
 
 # Posthog
 NEXT_PUBLIC_POSTHOG_KEY=""
-NEXT_PUBLIC_POSTHOG_HOST="https://us.i.posthog.com"
\ No newline at end of file
+NEXT_PUBLIC_POSTHOG_HOST="https://us.i.posthog.com"
+
+# AI Services & Infrastructure
+
+# Upstash Redis (for AI rate limiting and caching)
+# Get from: https://upstash.com/
+UPSTASH_REDIS_REST_URL="https://your-redis-url.upstash.io"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)